test/semantics/block-data01.f90

```diff
@@ -0,0 +1,13 @@
+block data foo
+  real :: pi = asin(-1.0) ! ok
```

Comment:
Would be nice again to write the clauses you are testing here.

Reply:
Done.